### PR TITLE
Time table

### PR DESCRIPTION
**Pull request by oleh-shev**
[ADD] new create method for new time tables;
[ADD] new method to format date (/shared/format-datepicker/format-datepicker.ts

### CODE REVIEW

File: src/app/admin/time-table/time-table-add-dialog/time-table-add-dialog.component.html [hunk 1]
```diff
@@ -10,15 +10,38 @@ <h3 mat-dialog-title>{{data.description.title}}</h3>
...
+      </mat-select>
+    </mat-form-field>
+  </div>
   <div>
     <mat-form-field>
-      <input matInput [matDatepicker]="picker" placeholder="Виберіть дату" disabled>
+      <input matInput [matDatepicker]="picker"
+             placeholder="Виберіть дату"
+             name="start_date" ngModel
+             #start_date="ngModel"
```
Comment (AndriyIf) [inline src/app/admin/time-table/time-table-add-dialog/time-table-add-dialog.component.html]: Are you sure that we need start_date ref? Why do we need `ngModel` without binding?

File: src/app/admin/time-table/time-table-add-dialog/time-table-add-dialog.component.html [hunk 2]
```diff
@@ -10,15 +10,38 @@ <h3 mat-dialog-title>{{data.description.title}}</h3>
...
+      <mat-error *ngIf="dateCtrl.errors?.required && dateCtrl.touched">Виберіть дату!</mat-error>
     </mat-form-field>
   </div>
-  <div class="ngx-timepicker-field-example">
-    <ngx-timepicker-field [format]="24"></ngx-timepicker-field>
+  <div>
+    <mat-form-field>
+      <input matInput placeholder="Виберіть годину"
+             name="start_time" ngModel
+             #start_time="ngModel"
```
Comment (AndriyIf) [inline src/app/admin/time-table/time-table-add-dialog/time-table-add-dialog.component.html]: Looks like start_time ref is not used as well.

File: src/app/admin/time-table/time-table-add-dialog/time-table-add-dialog.component.html (hunk 1 above)
Comment (oleh-shev) [reply src/app/admin/time-table/time-table-add-dialog/time-table-add-dialog.component.html]: I've removed unused ref and pushed commit changes to this branch

File: src/app/admin/time-table/time-table-add-dialog/time-table-add-dialog.component.html (hunk 2 above)
Comment (oleh-shev) [reply src/app/admin/time-table/time-table-add-dialog/time-table-add-dialog.component.html]: The same solution as in comment above

**Review by AndriyIf — APPROVED** [review]

**Review by yurakhomitsky — APPROVED** [review]

**Pull request merged**